### ISSUE TITLE
Broken: export-p12

### ISSUE THREAD

**Issue by petersm1** [OpenVPN/easy-rsa#183]
In 3.0.5 this is broken
/easyrsa  export-p12 client-name
Works on 3.0,4

It's extra quotation on $key_in, see below

//Peter

**3.0.5**
```
                # export the p12:
                "$EASYRSA_OPENSSL" pkcs12 -in "$crt_in" -inkey "$key_in" -export \
                        -out "$pkcs_out" "$pkcs_opts" || die "\
Export of p12 failed: see above for related openssl errors.
```

**3.0.4**
```
                # export the p12:
                "$EASYRSA_OPENSSL" pkcs12 -in "$crt_in" -inkey "$key_in" -export \
                        -out "$pkcs_out" $pkcs_opts || die "\
Export of p12 failed: see above for related openssl errors.

**Comment by Allfd** [comment]
Can confirm,  the quotes around $pkcs_opts are the problem.

**Comment by Antagonym** [comment]
This should be fixed as of 2ef26604.

**Comment by ecrist** [comment]
Fixed.  Closing.
